autogen/agentchat/conversable_agent.py

```diff
@@ -2521,13 +2523,14 @@ def _wrap_function(self, func: F) -> F:
         @functools.wraps(func)
         def _wrapped_func(*args, **kwargs):
             retval = func(*args, **kwargs)
-
+            log_function_use(self, func, kwargs, retval)
```

Comment:
This is not a function use event? It is more like a registering a function/tool event.

Reply:
Function use event should be inside `generate_tool_call_reply`

Reply:
this line is inside the wrapper, so the log will be called every time the function is called